packages/cli/tests/unit/depends.spec.ts

```diff
@@ -1,10 +1,11 @@
-const { utimesSync } = require('fs');
-const { join } = require('path');
-const tmp = require('tmp');
-const fs = require('fs-extra');
-const Depends = require('../../src/depends');
-const Fingerprinter = require('../../src/fingerprint/fingerprinter');
-const { listAppMapFiles, verbose } = require('../../src/utils');
+import { utimesSync } from 'fs';
+import { join } from 'path';
+import tmp from 'tmp';
+import fs from 'fs-extra';
+
+import Fingerprinter from '../../src/fingerprint/fingerprinter';
+import Depends from '../../src/depends';
+import { listAppMapFiles, verbose } from '../../src/utils';
```

Comment:
Please separate style commits from substantial ones. It makes it unnecessarily difficult to review the PR and use `git blame` later.